### DISCUSSION TITLE
buddhism/2021/confusion-to-clarity/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#62] [General]
# buddhism/2021/confusion-to-clarity/

https://shanenull.com/buddhism/2021/confusion-to-clarity/

**Comment by shane0** [comment]
# GUIDED MEDITATION ON IMPERMANENCE

   Sit in a relaxed posture with your back straight.
   Your eyes can be open or closed.
   Take a minute to rest in open awareness.
   Now choose an object that invites you to contemplate impermanence. You might choose your own body, or that of another person or a pet, or a house or a tree.
   As you examine this object, consider how it has already changed since its inception. Investigate how it changes from moment to moment.
   If you have chosen to contemplate your own impermanence, then consider asking:
Aren’t the years of my life slipping by?
Isn’t everything changing moment to moment?
Has anyone ever lived forever?
Isn’t it possible for death to come without warning?
   If you become distracted, just pause, and then continue.
   From time to time, drop the contemplation and rest in open awareness.
   Try this for five to ten minutes.
   Conclude by resting in open awareness.”

Excerpt From
Turning Confusion into Clarity
Yongey Mingyur
This material may be protected by copyright.

**Comment by shane0** [comment]
https://shanenull.com/buddhism/2023/draft/
